python_baseline_synthetic.py

```diff
@@ -0,0 +1,55 @@
+from river.forest import AMFClassifier
+from time import time
+import pandas as pd
+from river.tree.mondrian.mondrian_tree_nodes import MondrianLeafClassifier
+from river.tree.mondrian.mondrian_tree_nodes import MondrianBranchClassifier
+
+mf = AMFClassifier(
+    n_estimators=1,
+    use_aggregation=False,
+)
+
+df = pd.read_csv("/home/robotics/light-river/syntetic_dataset.csv")
```

Comment:
Do you think it is worth accessing this file directly from the URL, like you do on `synthetic.rs`?